netdisco/discoverables/yamaha.py

```diff
@@ -8,11 +8,14 @@ class Discoverable(SSDPDiscoverable):
     def info_from_entry(self, entry):
         """Return the most important info from a uPnP entry."""
         yam = entry.description['X_device']
+        services = yam['X_serviceList']['X_service']
+        if isinstance(services, list):
+            service = services[0]
```

Comment:
So, instead of this, we should probably do something like:

```
if isinstance(services, list):
   for s in services:
      if s['X_specType'] == 'urn:schemas-yamaha-com:service:X_YamahaRemoteControl:1':
         service = s